esti/gc_test.go

```diff
@@ -263,6 +262,28 @@ func prepareForGC(t *testing.T, ctx context.Context, testCase *testCase, blockst
 	return commitId
 }
 
+func getSparkSubmitArgs() []string {
+	return []string{
+		"--master", "spark://localhost:7077",
+		"--conf", "spark.driver.extraJavaOptions=-Divy.cache.dir=/tmp -Divy.home=/tmp",
+		"--conf", "spark.hadoop.lakefs.api.url=http://lakefs:8000/api/v1",
+		"--conf", "spark.hadoop.lakefs.api.access_key=AKIAIOSFDNN7EXAMPLEQ",
+		"--conf", "spark.hadoop.lakefs.api.secret_key=wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY",
+		"--class", "io.treeverse.clients.GarbageCollector",
+	}
+}
+
+func getDockerArgs(workingDirectory string) []string {
+	return []string{
+		"run", "--network", "host", "--add-host", "lakefs:127.0.0.1",
+		"-v", fmt.Sprintf("%s/ivy:/opt/bitnami/spark/.ivy2", workingDirectory),
+		"-v", fmt.Sprintf("%s:/opt/metaclient/client.jar", metaclientJarPath),
+		"--rm",
+		"-e", "AWS_ACCESS_KEY_ID",
+		"-e", "AWS_SECRET_ACCESS_KEY",
+	}
+}
```

Comment:
should this be here?

Reply:
nope - fixing invalid conflict resolution.